src/main/assembly/dist/cfg/application.properties

```diff
@@ -10,6 +10,11 @@ schemas.ddm=https://easy.dans.knaw.nl/schemas/md/ddm/ddm.xsd
 schemas.files=https://easy.dans.knaw.nl/schemas/bag/metadata/files/files.xsd
 schemas.agreements=https://easy.dans.knaw.nl/schemas/bag/metadata/agreements/agreements.xsd
 
+#
+# header used for http request such as XSD schema's
+#
+http.agent=easy-validate-dans-bag
```

Comment:
@janvanmansum For a moment I thought I had to configure this value in dtap. Do I?

Reply:
Not if you do not change the default.